### PR TITLE
feat: Update list tags structure

### PR DESCRIPTION
**Pull request by przemvs**
Update users list, devices  html tag to ul>li/

Change structure for right sidebar receipts.

### CODE REVIEW

File: src/script/components/panel/PanelActions/PanelActions.tsx [hunk 1]
```diff
@@ -39,6 +39,7 @@ const PanelActions: React.FC<PanelActionsProps> = ({items}) => {
     <ul css={listCSS}>
       {items.map(({click, identifier, icon, label}) => (
         <li key={identifier}>
+          1st one
```
Comment (PatrykBuniX) [inline src/script/components/panel/PanelActions/PanelActions.tsx]: is it expected? 😅

File: src/script/page/RightSidebar/ConversationDetails/components/ConversationDetailsOptions/ConversationDetailsOption.tsx [hunk 2]
```diff
@@ -43,8 +43,8 @@ const ConversationDetailsOption: FC<ConversationDetailsOptionProps> = ({
 }) => (
   <li className={className}>
     <button className="panel__action-item" onClick={onClick} data-uie-name={dataUieName} type="button">
+      2nd one
```
Comment (PatrykBuniX) [inline src/script/page/RightSidebar/ConversationDetails/components/ConversationDetailsOptions/ConversationDetailsOption.tsx]: is it expected?

File: src/script/components/panel/PanelActions/PanelActions.tsx (hunk 1 above)
Comment (przemvs) [reply src/script/components/panel/PanelActions/PanelActions.tsx]: Ahahaha.. forgot to remove it :D thanks for catch!

File: src/script/page/RightSidebar/ConversationDetails/components/ConversationDetailsOptions/ConversationDetailsOption.tsx (hunk 2 above)
Comment (przemvs) [reply src/script/page/RightSidebar/ConversationDetails/components/ConversationDetailsOptions/ConversationDetailsOption.tsx]: Ahahaha.. forgot to remove it :D thanks for catch!

**Review by PatrykBuniX — APPROVED** [review]

**Review by V-Gira — APPROVED** [review]

**Pull request merged**